go.mod

```diff
@@ -5,6 +5,6 @@ go 1.13
 require (
 	github.com/RoaringBitmap/roaring v0.4.23
 	github.com/blevesearch/bleve_index_api v1.0.0
-	github.com/couchbase/vellum v1.0.2
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
Interesting that this should add an indirect dependency.

Reply:
Wow, go modules continues to confuse.  I mean we're always running go mod tidy, and nothing else has changed... So WTF?

Reply:
Actually I have a theory...

Perhaps vellum and roaring both use go-spew, but vellum had it as a part of it's go.mod.  Now by removing vellum, there is no reference for it, so it becomes an indirect reference for our project???

Reply:
Oh wait, except we wrote vellum and have never heard of go-spew, lol

Reply:
Maybe something was cached weird for couchbase/vellum locally, so it never added it.  But when we switched to blevesearch/vellum it realized it needed it?

Reply:
That'd explain it, but vellum doesn't have a go-spew in it's list of direct dependencies.

Reply:
So looks like one of vellum's dependencies references it, it's part of vellum's go.sum. Your theory holds.